converter/src/main/java/gov/cms/qpp/conversion/encode/MeasureDataEncoder.java

```diff
@@ -53,14 +47,14 @@ protected void internalEncode(JsonWrapper wrapper, Node node) {
 	 *
 	 * @return intialized measure type map
 	 */
-	private Map<String, String> initializeMeasureTypeMap() {
-		Map<String, String> measureTypeMapper = new HashMap<>();
+	private Map<SubPopulationLabel, String> initializeMeasureTypeMap() {
+		Map<SubPopulationLabel, String> measureTypeMapper = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this Map to an EnumMap. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1640)